disk/block_test.go

```diff
@@ -107,7 +107,7 @@ func TestReadWriteBlock(t *testing.T) {
 		// write partly block
 		{90, 20, 4, 16},
 		// empty data
-		{100, 20, 0, 0},
```

Comment:
we can add the old {100, 20, 0, 0} test to test CRC error. a test case like {100, 20, 0} at line 83